Wordle.Api/Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class PlayerController : ControllerBase
+    {
+        private readonly PlayerService _playerService;
+
+        public PlayerController(PlayerService playerService)
+        {
+            _playerService = playerService;
+        }
+
+
+        [HttpGet("GetTopTenPlayers")]
```

Comment:
Typically, we don't add the action to the endpoint name.
```suggestion
        [HttpGet("TopTenPlayers")]
```